doc/usage/installation.rst

```diff
@@ -0,0 +1,147 @@
+=================
+Installing Sphinx
+=================
+
+.. contents::
+   :depth: 1
+   :local:
+   :backlinks: none
+
+
+Linux
+-----
+
+Debian/Ubuntu
+~~~~~~~~~~~~~
+
+Install ``python-sphinx`` using :command:`apt-get`:
+
+.. code-block:: bash
+
+   $ apt-get install python-sphinx
+
+If it not already present, this will install Python for you.
+
+RHEL, CentOS
+~~~~~~~~~~~~
+
+Install ``python-sphinx`` using :command:`yum`:
+
+.. code-block:: bash
+
+   $ yum install python-sphinx
+
+If it not already present, this will install Python for you.
+
+Other distributions
+~~~~~~~~~~~~~~~~~~~
+
+Most Linux distributions have Sphinx in their package repositories.  Usually
+the package is called ``python-sphinx`` or ``sphinx``.  Be aware that there are
+at least two other packages with ``sphinx`` in their name: a speech recognition
+toolkit (*CMU Sphinx*) and a full-text search database (*Sphinx search*).
+
+
+macOS
+-----
+
+Sphinx can be installed using `MacPorts <http://www.macports.org/>`_.
+
+.. code-block:: bash
+
+   $ sudo port install py27-sphinx
+
+To set up the executable paths, use the ``port select`` command:
+
+.. code-block:: bash
+
+   $ sudo port select --set python python27
+   $ sudo port select --set sphinx py27-sphinx
+
+Type :command:`which sphinx-quickstart` to check if the installation was
+successful.
+
+
+Windows
+-------
+
+.. todo:: Could we start packaging this?
```

Comment:
+0; if any newcomer works for this.

Anyway, is this todo needed on docs? I think an issue is good to us.

Reply:
I think so, yes. If we leave it here, hopefully someone will see it in the docs and work on it 😄

Reply:
Okay :-)